.changesets/feat_geal_jwks_headers.md

```diff
@@ -0,0 +1,5 @@
+### Add headers to the JWKS download request ([Issue #4651](https://github.com/apollographql/router/issues/4651))
+
+This adds the ability to set static headers on HTTP requests to download a JWKS from an identity provider
```

Comment:
The router supports the new `authentication.router.jwt.jwks.headers` option for setting static headers on HTTP requests to download a JWKS from an identity provider.

For details, see the [JWKS configuration option](https://www.apollographql.com/docs/router/configuration/authn-jwt#jwks).